testnet.docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+services:
+  backend:
+    container_name: backend
+    image: docker.io/desoprotocol/backend:latest
+    command: run
+    volumes:
+    - testnet-db:/db
+    ports:
+    - 18001:18001
+    - 18000:18000
+    - 2345:2345
+    environment:
+    # Values we set here OVERRIDE the base.env variables.
+    #
+    # Set the seed here if you want to be a validator
+    # - POS_VALIDATOR_SEED=
+    #
+    # For testnet, we must use a different checkpoint provider
+    - GLOG_V=1
+    - TRUSTED_BLOCK_PRODUCER_START_HEIGHT=2000000000
+    - FORCE_CHECKSUM=false
+    - SYNC_TYPE=blocksync
+    - CHECKPOINT_SYNCING_PROVIDERS=https://test.deso.org
+    - TESTNET=true
+    # The REGTEST param is the main difference between running a testnet vs devnet node.
+    - REGTEST=false
+    - CONNECT_IPS=dorsey.bitclout.com:18000
+    - PROTOCOL_PORT=18000
+    - API_PORT=18001
+    env_file:
+    - base.env
+    expose:
+    - "18001"
+    - "18000"
+    - "2345"
+  frontend:
+    container_name: frontend 
+    image: docker.io/desoprotocol/frontend:stable
```

Comment:
stable or latest? we should just be consistent.

Reply:
Stable everywhere now.